en/flight_controller/snapdragon_flight_software_installation.md

```diff
@@ -0,0 +1,215 @@
+# Installing Software on the Snapdragon
+To get the platform running with the complete VIO system running, multiple pieces are required. The following is an overview of the different versions used.
+
+| Software	| Tested version			| Link/commit ID				|
+|---------------|---------------------------------------|-----------------------------------------------|
+| ROS		| indigo				|[here](http://wiki.ros.org/indigo)		|
+| MAVROS	| 0.19.0				|[399a04e](https://github.com/nicovanduijn/mavros/commit/399a04ef5739bd93fe1c596b63d6bdbfba7d84ef)|
+| MAVLink	| release/kinetic/mavlink/2017.5.5-0 	|[2ec76d2](https://github.com/mavlink/mavlink-gbp-release/commit/2ec76d25c6b7e6fac1f6b0d08765d07aa0f4f69d)|
+| PX4		| 1.7.3 + additions			|[087f0f8](https://github.com/ChristophTobler/Firmware/commit/087f0f8bf014a6b073c12408f35cb9fb2e588af9)|
+| Snap VIO	| mv-1.0.2/mavros-integration		|[eecc3c2](https://github.com/nicovanduijn/snap-vislam-ros/commit/eecc3c24cb2dc4451df1e97f699fffdaac04f266)|
+| Snapdragon-Linux| 3.1.3.1				|[here](https://support.intrinsyc.com/projects/snapdragon-flight/files)|
+| DSP Firmware	| 3.1.3.1				|[here](https://support.intrinsyc.com/projects/snapdragon-flight/files)|
+| Qualcomm MV	| 1.0.2					|[here](https://developer.qualcomm.com/sdflight-tools)|
+
+
+## Prevent bricking
```

Comment:
First letter capitalization everywhere in headings. i.e. Prevent Bricking.

Reply:
done